docs/README.md

```diff
@@ -1,42 +1,55 @@
-- *Grafana*: The *Explore* feature of Grafana 6.0+ is the primary place of
-  contact between a human and Loki. It is used for discovering and analyzing
-  logs.
+Unlike other logging systems, Loki is built around the idea of only indexing
+labels for logs and leaving the original log message unindexed. This means
+that Loki is cheaper to operate and can be orders of magnitude more efficent.
 
-Alongside these main components, there are some other ones as well:
+## Table of Contents
 
-- **[LogCLI](logcli.md)**: A command line interface to query logs and labels
-  from Loki
-- **[Canary](canary/README.md)**: An audit utility to analyze the log-capturing
-  performance of Loki. Ingests data into Loki and immediately reads it back to
-  check for latency and loss.
-- **[Docker
-  Driver](https://github.com/grafana/loki/tree/master/cmd/docker-driver)**: A
-  Docker [log
-  driver](https://docs.docker.com/config/containers/logging/configure/) to ship
-  logs captured by Docker directly to Loki, without the need of an agent.
-- **[Fluentd
-  Plugin](https://github.com/grafana/loki/tree/master/fluentd/fluent-plugin-grafana-loki)**:
-  An Fluentd [output plugin](https://docs.fluentd.org/output), to use Fluentd
-  for shipping logs into Loki
+1. [Overview](overview/README.md)
+    1. [Comparison to other Log Systems](overview/comparisons.md)
+2. [Installation](installation/README.md)
+    1. [Installing Locally](installation/local.md)
+    2. [Installing with Helm](installation/helm.md)
+    3. [Installing with Ksonnet](installation/ksonnet.md)
+3. [Getting Started](getting-started/README.md)
+    1. [Grafana](getting-started/grafana.md)
+    2. [Logcli](getting-started/logcli.md)
+    4. [Troubleshooting](getting-started/troubleshooting.md)
+4. [Configuration](configuration/README.md)
+    1. [Examples](configuration/examples.md)
+5. [Clients](clients/README.md)
+    1. [promtail](clients/promtail/README.md)
+        1. [Installation](clients/promtail/installation.md)
+        2. [Configration](clients/promtail/configuration.md)
+        3. [Scraping](clients/promtail/scraping.md)
+        4. [Pipelines](clients/promtail/pipelines.md)
+        5. [Troubleshooting](clients/promtail/troubleshooting.md)
+    2. [Docker Driver](clients/docker-driver/README.md)
+        1. [Configration](clients/docker-driver/configuration.md)
+    3. [fluentd](clients/fluentd.md)
+    4. [fluent-bit](clients/fluent-bit.md)
+6. [LogQL](logql.md)
+7. [Operations](operations/README.md)
+    1. [Authentication](operations/authentication.md)
+    2. [Observability](operations/observability.md)
+    3. [Scalability](operations/scalability.md)
+    4. [Storage](operations/storage/README.md)
+        1. [Table Manager](operations/storage/table-manager.md)
+        2. [Retention](operations/storage/retention.md)
+    5. [Multi-tenancy](operations/multi-tenancy.md)
+    6. [Loki Canary](operations/loki-canary.md)
+8. [HTTP API](api.md)
+9. [Architecture](architecture.md)
```

Comment:
Since architecture cover multi-tenancy, storage and so on, better to move it before the Operations chapter?

Reply:
The Architecture section is meant just to have extra details for the curious (or potential maintainers) and shouldn't be a required read, which is why it's at the end.